CMakeLists.txt

```diff
@@ -29,20 +29,24 @@ cmake_minimum_required(VERSION 3.11)
 
 set(PROJECT smartDeviceLinkCore)
 project(${PROJECT})
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH}
-    "${CMAKE_SOURCE_DIR}/tools/cmake/modules/")
+set(
```

Comment:
@MaximRybak bring to the form as
```
set(CMAKE_MODULE_PATH
      ${CMAKE_MODULE_PATH}
      "${CMAKE_SOURCE_DIR}/tools/cmake/modules/"
)
```

Reply:
CMAKE_MODULE_PATH this is a variable which include list of argument. This is a correct formatting for variable with list argument. https://cmake.org/cmake/help/latest/variable/CMAKE_MODULE_PATH.html

Reply:
@MaximRybak  it is bad formatting, please fix it.

Reply:
@MaximRybak It is bad formatting. please fix it